### PR TITLE
fix keyerror on order placement

### PR DESCRIPTION
**Pull request by MaxxRK**
If an account cannot place an order it will cause a KeyError in some instances. This just accepts the error and moves on.

### CODE REVIEW

File: tastyAPI.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from decimal import Decimal as D
 
 from dotenv import load_dotenv
+from helperAPI import Brokerage, maskString, printAndDiscord, printHoldings, stockOrder
```
Comment (NelsonDane) [inline tastyAPI.py:10]: This doesn't need to move to a different spot

**Review by NelsonDane — CHANGES_REQUESTED** [review]
Looks good except for the import order changing

File: tastyAPI.py (hunk 1 above)
Comment (MaxxRK) [reply tastyAPI.py:10]: I agree

**Review by NelsonDane — APPROVED** [review]
LGTM

**Pull request merged**